schema/deploy/shipping/views.sql

```diff
@@ -4206,6 +4207,7 @@ create or replace view shipping.linelist_data_for_wa_doh_v1 as (
       -- `shipping.return_results_v3` uses this same filter.
       where collected >= '2020-06-10 00:00:00 US/Pacific'
       and hcov19_presence_absence_result_v1.details @> '{"assay_type": "Clia"}'
+      and not sample.details @> '{"note": "never-tested"}'::jsonb
```

Comment:
Looks good to me. With current prod data, I see this filtering out 1 result. Out of curiosity, why the explicit cast to jsonb?

Reply:
I ended up deleting the P/A records for that one sample.

Casting to jsonb may not be necessary, but that's how it's shown in postgres docs: https://www.postgresql.org/docs/13/functions-json.html#FUNCTIONS-JSONB-OP-TABLE